forem/forem
# Minor accessibility fixes for the podcast area

## What type of PR is this? (check all applicable)


## Description

This PR sweeps up some small accessibility bug fixes in the podcasts area. Specifically:

- Converts the main page heading to a `h1`
- Converts the sidebar to a navigation landmark containing a list of the podcasts
- Removes the `header` element (since pages should only have one header, and that's our top sticky header)
- Hides decorative images from screen readers where we were previously repeating the podcast name twice in a link

There are other accessibility issues remaining in this area, but they mostly either tie in to wider "Feed" issues, or to the podcast playback bar at the bottom of the screen which I've added a separate issue for: https://github.com/forem/forem/issues/14607 

## Related Tickets & Documents

N/A

## QA Instructions, Screenshots, Recordings

- Visit the `/pod` page and check visually everything is as before

Checking with a screen reader, you should be able to see that:
- The sidebar links are now in a nav with list items
- Link names are no longer e.g. "CodeNewbie CodeNewbie" but named just once 😄 

![screenshot showing VoiceOver on the podcast page announcing the podcasts navigation](https://user-images.githubusercontent.com/20773163/131118797-d7a74433-e040-4d3d-a3b0-28a43875f567.png)
![screenshot showing VoiceOver on the podcast page announcing the CodeNewbie link](https://user-images.githubusercontent.com/20773163/131118811-b5620d4a-79cf-4374-a2c4-05c286cdad57.png)


### UI accessibility concerns?

Fixes some minor issues

## Added/updated tests?

- [X] No, and this is why: Existing tests continue to check the appropriate links etc are in place

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Admin Guide](https://admin.forem.com/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
- [X] This change does not need to be communicated, and this is why not: these are quite minor changes, and there's still work to do in this area in terms of accessibility. I don't think it merits any communication of its own
